src/module/common/button.tsx

```diff
@@ -1,28 +1,109 @@
-import { cva, type VariantProps } from "class-variance-authority";
-import React from "react";
+import { cva, type VariantProps } from 'class-variance-authority'
+import React from 'react'
 
 export interface ButtonVariants extends VariantProps<typeof buttonVariants> {}
-const buttonVariants = cva(["btn"], {
-  variants: {
-    intent: {
-      default: [],
-      outline: ["btn-outline"],
+const buttonVariants = cva(
+  ['btn', 'overflow-hidden', 'relative', 'rounded', 'border','transition-all','duration-500'],
```

Comment:
can remove `btn` class since we not using daisy anymore.

can you turn this into a string btw? string[] is a bit painful to watch! hahah